Makefile

```diff
@@ -280,8 +280,8 @@ container: buildkitd
 	@docker load < build/$@.tar
 
 .PHONY: basic-integration
-basic-integration:
-	@TAG=$(TAG) ./hack/test/$@.sh
+basic-integration: talos-test
+	@TAG=$(TAG) $(BINDIR)/talos-test basic-integration
```

Comment:
Instead of building it, I maybe it might be better to just `go run`? That way we don't have to add the build to drone at all.

Reply:
had the same thought and it makes my life easier anyways. will move this over.

Reply:
fixed in amended commit.